src/languages/en_US.json

```diff
@@ -1228,6 +1229,10 @@
         "address_copied_to_clipboard": "Address copied to clipboard"
       }
     },
+    "points": {
+      "coming_soon_title": "Coming Soon",
+      "coming_soon_description": "Points are just around the corner.\nStart earning today."
```

Comment:
I think this copy is a little confusing personally. What actions can the user take `today` to start earning?

Reply:
+1, what actions can I take today? 🌈